tests/plugins/widget/dnd.js

```diff
@@ -54,11 +54,12 @@
 	var getWidgetById = widgetTestsTools.getWidgetById,
 		assertRelations = lineutilsTestsTools.assertRelations;
 
-	function dragstart( editor, evt, widget ) {
-		var dropTarget = CKEDITOR.plugins.clipboard.getDropTarget( editor );
+	function dragstart( editor, evt, widgetOrNode ) {
```

Comment:
This change doesn't seem very elegant :thinking: Maybe you could extract this as 2 helpers instead, so whole logic would be encapsulated in one function handling node elements + helper searching for drag handler container inside widget?

The same goes to `dragend` function.